qa/tasks/cephfs_test_runner.py

```diff
@@ -133,7 +133,9 @@ def task(ctx, config):
 
     # Mount objects, sorted by ID
     if hasattr(ctx, 'mounts'):
-        mounts = [v for k, v in sorted(ctx.mounts.items(), lambda a, b: cmp(a[0], b[0]))]
+        mounts = [v for k, v in sorted(
+            ctx.mounts.items(),
+            lambda a, b: (a[0] > b[0]) - (a[0] < b[0]))]
```

Comment:
can we use the python3 compatible way?
```py
mounts = [v for k, v in sorted(ctx.mounts.items(), key=lambda mount: mount[0])]
```

Reply:
Missed that one. Thanks for the hint.
Done